src/commands/download_compiler.rs

```diff
@@ -0,0 +1,23 @@
+use eyre::ContextCompat;
+
+/*
+mkdir -p 'src/compiler/bin' \
+    && wget 'https://github.com/matter-labs/zksolc-bin/raw/main/macosx-arm64/zksolc-macosx-arm64-v1.3.14' -O 'src/compiler/bin/zksolc' \
+    && chmod +x 'src/compiler/bin/zksolc' \
+    && brew install solidity \
+    && cp /opt/homebrew/bin/solc src/compiler/bin/solc
+*/
```

Comment:
I think that this is not working as expected, when I ran `zksync-era-cli download-compiler` it creates a bunch of directories named: `&&`, `brew`, `chmod`, `wget`, etc... maybe all in `args` function are being parsed as arguments for `mkdir`